packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newLineNameMorphFor..st

```diff
@@ -0,0 +1,10 @@
+private
+newLineNameMorphFor: aTraceValue
+	
+	| name |
+	name := aTraceValue belongingProbeId asString asText.
```

Comment:
why the additional `asText`?

Reply:
in general, why doesn't a String suffice?

Reply:
New Solution is not using a Text anymore